src/index.js

```diff
@@ -3,14 +3,21 @@ import { Provider } from 'react-redux';
 import ReactDOM from 'react-dom/client';
 
 import './index.css';
+import Api from 'process/api';
 import Application from './Application';
-import store from './process/redux/store';
+import { store, persistedStore } from './process/redux/store';
+import { PersistGate } from 'redux-persist/integration/react';
```

Comment:
Last comment on this PR, if you solve it just merge this, don't wait for my approval.

We group those by: 
* external libraries
* absolute paths
* relative paths

Think about this, do the proper thing.